### DISCUSSION TITLE
Revision Date Icon - How'd You Do It?

### DISCUSSION THREAD

**Discussion by tmeuze** [squidfunk/mkdocs-material#4456] [Q&A] [answered]
Hi squidfunk, 

Just curious how you were able to change the "Last update" text to an icon, using the `mkdocs-git-revision-date-localized-plugin` (on the documentation site)?

![CleanShot 2022-10-04 at 16 26 54](https://user-images.githubusercontent.com/9298659/193920115-15787058-8192-48aa-b93c-2ee433e06655.png)

Thank you!

**Comment by squidfunk** [comment, marked as answer]
It's included with Insiders and essentially a refactoring of the [revisioning](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#revisioning) functionality which will be released when [document contributors](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#document-contributors) will be released 😀 Other than hat, Insider access will get you the new styles immediately.

**Reply by tmeuze** [reply]
Very neat! Thank you @squidfunk
